NOnion/Cells/Relay/CellPlainRelay.fs

```diff
@@ -32,6 +32,7 @@ type RelayData =
         | 2uy -> RelayData data
         | 3uy -> RelayEnd (reader.ReadByte ())
         | 4uy -> RelayConnected data
+        | 9uy -> reader.ReadByte () |> RelayTruncated
```

Comment:
I think the point of using `[<Literal>]` in Command.fs is so that here you don't need to hardcode this magic numbers, just use the constants there?